docs/posts/prepare.md

```diff
@@ -0,0 +1,104 @@
+
+## Example usage
+
+As a quick example, let's construct a command to use `prepare` to take a raw expression matrix and generate a processed `h5ad` ready to visualize with cellxgene.
+
+We'll start off using the raw data from the pbmc3k dataset. This dataset is described [here](https://icb-scanpy.readthedocs-hosted.com/en/stable/api/scanpy.datasets.pbmc3k.html), and is available as part of the scanpy package. For this example, we'll assume this raw data is stored in a file called `pbmc3k-raw.h5ad`.
+
+Our `prepare` compose our command looks like this:
+
+```
+cellxgene prepare pbmc3k-raw.h5ad \
+	--run-qc \                                  # (A)
+	--recipe seurat \                           # (B)
+	--layout tsne --layout umap \               # (C)
+	--output pbmc3k-prepared.h5ad               # (D)
+```
+
+Let's look at what `prepare` is doing to our data, and how each step relates to the command above. You can see a walkthrough of what's going on under the hood for this example in [this notebook](https://github.com/chanzuckerberg/cellxgene-vignettes/blob/master/dataset-processing/pbmc3k-prepare-example.ipynb).
+
+**(A) - Compute quality control metrics and store this in our `AnnData` object for later inspection**   
+**(B) - Normalize the expression matrix using a basic preprocessing recipe**   
+**(auto) - Do some preprocessing to run PCA and compute the neighbor graph**  
+**(auto) - Infer clusters with the Louvain algorithm and store these labels to visualize later**  
+**(C) - Compute and store umap and tsne embeddings**  
+**(D) - Write results to file**
+
+## Options for cellxgene `prepare`
+
+**For the most up-to-date and comprehensive list of options, run `cellxgene prepare --help`**
+
+`--embedding` controls which dimensionality reduction algorithm is applies to your data.  
+Options are `umap` and/or `tsne`. Defaults to both.
+
+`--recipe` controls which normalization steps to apply to your data, based on one of the preprocessing `recipes` included with `scanpy`.
+These recipes include steps like cell filtering and gene selection; see the `scanpy` [documentation](https://scanpy.readthedocs.io/en/latest/api/index.html#recipes) for more details.   
+Options are `none`, `seurat`, or `zheng17`. Defaults to `none`.
+
+`--sparse` is a flag determines whether to enforce a sparse matrix. For large datasets, `prepare` can take a long time to run (a few minutes for datasets with 10-100k cells, up to an hour or more for datasets with >100k cells). If you want `prepare` to run faster we recommend using the `sparse` option.  
+If this flag is not included, default is `False`
+
+`--run-qc / --skip-qc` is a flag that determines whether to calculate QC metrics (saved to `anndata.obs` and `anndata.var`). See the `scanpy` [documentation](https://scanpy.readthedocs.io/en/stable/api/scanpy.pp.calculate_qc_metrics.html) for details.
```

Comment:
FYI - changing in 0.13.   It will be a simple flag called `--skip-qc`.  By default, qc metrics will be generated.